<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cooper Black</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selamat Datang di Toko RISNI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hapus">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ubah">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>UBAH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Berikut adalah Tabel Daftar Rokok yang tersedia di Toko Risni</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tambah">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>TAMBAH</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_rokok">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>521</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Code</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nama</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Harga</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Stok</string>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sumber : Risnianti Nurkhalifah_2210010104_5B Nonreg Bjb T.Informatika_Visual3</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDAFTAR_ROKOK_RISNI">
    <property name="title">
     <string>DAFTAR ROKOK RISNI</string>
    </property>
   </widget>
   <addaction name="menuDAFTAR_ROKOK_RISNI"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
